libbeat/_meta/config.yml.tmpl

```
@@ -85,15 +85,25 @@ output.elasticsearch:
 
   # Client Certificate Key
   #ssl.key: "/etc/pki/client/cert.key"
-
+{{end}}
 #================================ Processors =====================================
-
+{{if not .UseObserverProcessor}}
 # Configure processors to enhance or manipulate events generated by the beat.
 
 processors:
   - add_host_metadata: ~
   - add_cloud_metadata: ~
-
+{{else}}
+processors:
+  - add_observer_metadata: 
+  # Optional, but recommended geo settings for the location heartbeat is running in 
```

Also this :slightly_smiling_face: 
```suggestion
  # Optional, but recommended geo settings for the location {{ .BeatName }} is running in 
```